roles/dugtrio/defaults/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+
+# Set this to true if you want to stop everything and wipe the databases
+dugtrio_cleanup_all: false
+
+
+dugtrio_user: dugtrio
+dugtrio_datadir: "/data/dugtrio"
+dugtrio_docker_network_name: shared
+dugtrio_docker_networks:
+  - name: "{{ dugtrio_docker_network_name }}"
+
+# dugtrio config
+dugtrio_endpoints:
+  - name: "default"
+    url: "http://your-cl-node:5052"
+dugtrio_pool_scheduler: "rr"
+dugtrio_pool_follow_distance: 10
+dugtrio_pool_max_head_distance: 2
+dugtrio_proxy_count: 1
+dugtrio_proxy_call_timeout: "55s"
+dugtrio_proxy_session_timeout: "10m"
+dugtrio_proxy_sticky_endpoint: true
+dugtrio_proxy_call_rate_limit: 100
+dugtrio_proxy_call_rate_burst: 1000
+dugtrio_proxy_blocked_paths: []
+dugtrio_frontend_enabled: true
+dugtrio_frontend_title: "Dugtrio"
+dugtrio_metrics_enabled: true
+
+# ------------------------------------------------------------------
+
+# dugtrio
+dugtrio_enabled: true
+dugtrio_container_name: dugtrio
+dugtrio_container_image: ethpandaops/dugtrio:1.0.0
+dugtrio_container_env: {}
+dugtrio_container_ports: []
+dugtrio_container_volumes:
+  - "{{ dugtrio_datadir }}:/config"
+dugtrio_container_stop_timeout: "600"
+dugtrio_container_pull: false
+dugtrio_container_networks: "{{ dugtrio_docker_networks }}"
+dugtrio_container_command: -config=/config/dugtrio.yaml
+
+dugtrio_config: |
```

Comment:
looks like there is a jinja spacing warning from the ci. Can you pls fix this?

Reply:
fixed :)